### PR TITLE
Rename listview.md to collectionview.md and update its terminology

### LINKED ISSUE #6553: Collection (formerly ListView) Article for v14+ needs updated terminology
## What type of issue is it?
Wrong documentation

## What article/section is this about?
https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/listview

## Describe the issue
While I was working on umbraco/Umbraco-CMS#17230 for Hacktoberfest, I discovered that the documentation for the Collection property type needs updating to make sure it uses the terminology for current versions of Umbraco.  If I understand correctly, the mentions of ListView should be changed to Collection and references to Content App should be changed to Workspace View.

If it's helpful as a point of reference, I've updated the Backoffice elements in umbraco/Umbraco.CMS.Backoffice#2452.  I'm not too committed to any specific verbiage, so I'm happy to update my PR there with different text if it helps ensure all of these bits and pieces are consistent.  😄 

I _think_ the change from List View to Collection happened as of v14, but I'm not 100-percent certain.  🤔 

I would love to submit a PR for this as part of my work on the issue above, but I don't know if I will have an opportunity.  With that in mind, I thought I would create this issue in case anyone participating in Hacktoberfest had a chance to snag it.  😄

### PR DESCRIPTION
**Pull request by caronvincent**
## Description

_What did you add/update/change?_

The aim of this pull request is to update the contents of the List View documentation page to use the updated terminology from Umbraco CMS 14, where "List view" is now known as "Collection view" and "Content App" is now known as "Workspace View".

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco CMS >=14

## Deadline (if relevant)

_When should the content be published?_

N/A

## Notes

The Alias has not been changed from `Umbraco.ListView` as it appears to still be in use in the backoffice.
Some screenshots will eventually need to be updated when the backoffice uses the new terminology as well.

## Related issue

Fixes #6553

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @caronvincent ! 💪 

I noticed from the UI in a 14 project, that the Property Editor is called "Collection" and not "Collection View" 🤔 
Looks like it's the UI schema that's called Collection View.

image

I think we should update it to "Collection" 😄 You up for it?

Also, you mention in your description, that you're also making some changes to Content App vs Workspace View.
It don't see any files with changes related to that? 🤔

**Comment by caronvincent** [comment]
@sofietoft Thank you for the feedback.

> I noticed from the UI in a 14 project, that the Property Editor is called "Collection" and not "Collection View" 🤔
Looks like it's the UI schema that's called Collection View.
I think we should update it to "Collection" 😄 You up for it?

To be clear on what you'd like me to do: since the backoffice refers to "Collection Property Editor" and not "Collection View Property Editor", all mentions of "Collection View" that I made in my PR should be changed to just "Collection" instead?

>Also, you mention in your description, that you're also making some changes to Content App vs Workspace View.
It don't see any files with changes related to that? 🤔

In collectionview.md (formerly listview.md), all mentions of "Content App" have been replaced by "Workspace View".
The changes in my PR so far are limited to collectionview.md as that was the file designated in issue #6553, however I can confirm that other files in the documentation refer to "content app".
Would you like me to expand the scope of this PR to change these other files as well?

**Comment by sofietoft** [comment]
Yes, please replace all "Collection View"s with "Collection".

Thanks for clearing up about the Content App mentions!
I think the scope of this PR is fine for now.
You're more than welcome though, to do another where you replace the Content App references 🙌

**Comment by caronvincent** [comment]
Understood, I will update mentions of "Collection View" in favor of "Collection".

I will limit this PR to this particular file, but it may be a good idea to create a new issue asking contributors to update mentions of "Content App" in favor of "Workspace View" across the entire documentation. It would make for an easy Hacktoberfest grab, much like the issue I am addressing with this PR.

**Comment by sofietoft** [comment]
That is very true and a great idea.
I'll add an issue to the tracker 🙌

**Comment by caronvincent** [comment]
I have pushed commit [f7206cc](https://github.com/umbraco/UmbracoDocs/pull/6571/commits/f7206cc880e036c04979330ed6528c8b9e0f3810) to fix the terminology as discussed.

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/collection.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+You can select order of the content nodes displayed, "Acsending" or "Descending". The order is affected by the "Order By" selection.
+
+### Columns Displayed
+
+It is possible to add more columns to the collection, via adding the properties through the dropdown. These properties are based on the Data Types which are used by the Document Type. It will show up in the dropdown by its alias and not the name on the property.
+
+![Collection property example](../../../../../../10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-property-editors/images/listview-property.png) ![Collection property example](../../../../../../10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-property-editors/images/listview-property-dropdown.png)
+
+Once you have selected a column that you want to display, the next thing you want to do is define what its name should be and what kind of value it should display. You can also move the headers around, re-ordering how the headers should look. This is done by the move icon on the left side of the alias.
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/collection.md:46]:
```suggestion
Once you have selected a column you want to display, define what its name should be and what kind of value it should display. You can also move the headers around, re-ordering how they should look. This is done by the move icon on the left side of the alias.
```

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/collection.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+![Collection content example email](../../../../../../10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-property-editors/images/listview-content-example-email.png)
+
+### Member name
+
+First, a Member Picker property needs to be present on the content item. In this example, the `child item` has gotten a Member Picker Data Type with the alias of `isAuthor`.
+
+![Collection member picker](../../../../../../10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-property-editors/images/member-picker.png)
+
+Now that the child item has a member and the value that should be displayed is the name of the picked value, the next step is to reconfigure the template value in the collection setting.
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/collection.md:100]:
```suggestion
The child item has a member and the value that should be displayed is the name of the picked value. The next step is to reconfigure the template value in the collection setting.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @caronvincent 🙌 

I've added a couple of suggestions to shorten a couple of sentences.
Also, we'll need a redirect for the renamed article.

The following line should be added to the bottom of the redirect list in the `.gitbook.yaml` file within the `14/umbraco-cms` directory:

```
fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/listview: fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/collection.md
```

**Comment by caronvincent** [comment]
Done!

**Comment by sofietoft** [comment]
Excellent! Let's get this one merged then 😄 

Thanks again for making this PR helping us getting this up to date 💪 MUCH appreciated!

**Pull request merged**